### PR TITLE
Update to bytes 0.5 and amino_rs 0.5

### PR DESCRIPTION
**Pull request by zmanian**
Update to bytes_0.5 and amino_rs 0.5 and makes all the amino_rs tokens independent of prost tokens enabling prost and amino_rs to be combined in a dependency.

### CODE REVIEW

File: tendermint/src/lib.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ extern crate prost_amino as prost;
 #[allow(unused_extern_crates)]
 #[macro_use]
 extern crate prost_amino_derive as prost_derive;
+extern crate bytes;
```
Comment (tarcieri) [inline tendermint/src/lib.rs]: Are either of these `extern crate`s still needed? I know `prost` was looking directly in the `sysroot` for them before which precluded renaming them, but if everything is namespaced correctly, it seems like it should be possible to get rid of them?

**Comment by Liamsi** [comment]
I assume this was tested locally, as well as running clippy on it? CircleCi doesn't run on forks :-/

File: tendermint/src/lib.rs (hunk 1 above)
Comment (Liamsi) [reply tendermint/src/lib.rs]: @xla might have addressed some of it here: https://github.com/interchainio/tendermint-rs/pull/122

Comment (Liamsi) [reply tendermint/src/lib.rs]: https://github.com/interchainio/tendermint-rs/pull/127

**Comment by ebuchman** [comment]
> CircleCi doesn't run on forks :-/

Fixed

**Pull request merged**